BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -292,6 +299,14 @@ public class MissileLauncher : MissileBase
 
         private bool OldInfAmmo = false;
         private bool StartSetupComplete = false;
+        //Fuel Burn Variables
+        public float GetModuleMass(float baseMass, ModifierStagingSituation situation) => ordinanceMass;
+        public ModifierChangeWhen GetModuleMassChangeWhen() => ModifierChangeWhen.CONSTANTLY;
+
+        private float burnRate;
+        private float burnedMass;
+        private float ordinanceMass;
```

Comment:
this should be ``burnedMass = 0;`` and ``ordinanceMass = 0;``

Reply:
`float`s default to 0, so that shouldn't actually be a problem.